requirements/base.txt

```diff
@@ -0,0 +1,5 @@
+lxml==4.6.4
```

Comment:
Hello ! I think we will have to use this file inside the dockerfile because the dependancies are outdated now and not sync with those requirements. ^^'

Reply:
I think it's easy to do but if you prefer, we could just trace this action and do it latter. :)

Reply:
A new PR should be available to review docker images (see issue https://github.com/IGNF/route-graph-generator/issues/21). This will be done here.

Reply:
I tried this 
```
# Comment the first pip install ...
COPY requirements ../
RUN cd ../ && python -m pip install -U -r base.txt && pip install -e .
``` 
And it worked. :)

Reply:
> A new PR should be available to review docker images (see issue #21). This will be done here.

ok !